### PR TITLE
Integration test: making it cooler!

### PR DESCRIPTION
**Pull request by lemunozm**
# Description

```rust
#[test_runtimes([development, altair, centrifuge])]
fn foo<T: Runtime> {
    // Your test here...
}
```
or 
```rust
#[test_runtimes(all)]
fn foo<T: Runtime> {
    // Your test here...
}
```

## Benefits
- It's prettier!
- Better readability 
- We no longer confuse an utility method with the same "shape" as foo from an actual test method

### CODE REVIEW

File: runtime/integration-tests/src/generic/cases/example.rs [hunk 1]
```diff
@@ -70,6 +71,7 @@ fn transfer_balance<T: Runtime>() {
 }
 
 // Identical to `transfer_balance()` test but using fudge.
+#[test_runtimes([development, altair, centrifuge])]
```
Comment (wischli) [inline runtime/integration-tests/src/generic/cases/example.rs:74]: Nit:

```suggestion
#[test_runtimes(all)]
```

**Review by wischli — DISMISSED** [review]
Awesome improvement 🚀

File: runtime/integration-tests/src/generic/cases/example.rs [hunk 2]
```diff
@@ -16,6 +16,7 @@ use crate::{
 	utils::accounts::Keyring,
 };
 
+#[test_runtimes([development, altair, centrifuge])]
```
[the remark this answers is not in the document]
Comment (lemunozm) [reply runtime/integration-tests/src/generic/cases/example.rs:19]: Yeah! I left this on purpose as an example of different runtimes you can set (not that this belongs to `example.rs`)

File: runtime/integration-tests/src/generic/cases/example.rs (hunk 1 above)
Comment (lemunozm) [reply runtime/integration-tests/src/generic/cases/example.rs:74]: Same as before

**Review by mustermeiszer — DISMISSED** [review]
🚀

**Comment by lemunozm** [comment]
@wischli @mustermeiszer @cdamian Can I have a reapproval?

**Pull request merged**